readme.txt

```diff
@@ -243,6 +243,23 @@ Our Cookie and Consent Banner can help you comply with GDPR. The European Union
 
 == Changelog ==
 
+= 9.0.2 =
+
+* Release date: October 9, 2020
+
+**Enhancements**
+
+* Publicize: improve handling of URLs when generating Twitter threads.
+
+**Bug fixes**
+
+* Instagram Embeds: ensure that Instagram URLs with additional URL parameters can be embedded as well.
+* Media Extractor: prevent PHP notice for some URLs.
+* Publicize: avoid block editor errors when no Publicize connections are available.
+* Synchronization flow: avoid potential Fatal errors when updating the plugin from the Dashboard > Updates screen.

```

Comment:
It strikes me as odd that "Fatal" is capitalized here.

Reply:
Updated in 069442665cd3f5b5c3ad1aece9f52821d3e610a8